### DISCUSSION TITLE
How can I deal with this problem

### DISCUSSION THREAD

**Discussion by JasonKe1031** [laravel/framework#45378] [General]
![image](https://user-images.githubusercontent.com/68834574/208874645-abd6b1ab-ed05-4e10-8325-e2efd928fe33.png)

**Comment by henzeb** [comment]
Well, I don't know, there is not much to go on. If admin.blade.php is yours, you should look for a property that might not be set. use xdebug. Check what variables it is expecting and see if they are truly being set.

But please, if you have a problem, that you cannot solve, tell us what you did do, tell us enough details so we can actually help you.
